ports/servoshell/desktop/app.rs

```diff
@@ -48,6 +49,7 @@ pub struct App {
     t_start: Instant,
     t: Instant,
     state: AppState,
+    webdriver_receiver: Option<Receiver<WebDriverCommandMsg>>,
```

Comment:
Apologies, I missed one comment. Please move this to `RunningAppState`, which will make it more obvious what its lifetime is.